.readthedocs.yml

```
@@ -4,6 +4,15 @@ build:
     os: "ubuntu-22.04"
     tools:
         python: "mambaforge-23.11"
+    jobs:
+        # Sometimes RTD only fetches a commit and not a tag.
+        # https://github.com/readthedocs/readthedocs.org/issues/10715
+        post_checkout:
+            - git fetch --tags
+        # Avoid a dirty index:
+        # https://docs.readthedocs.com/platform/latest/build-customization.html#avoid-having-a-dirty-git-index
+        pre_install:
+            - git update-index --assume-unchanged environment.yml docs/conf.py
```

```suggestion
            - git update-index --assume-unchanged environment.yml docs/source/conf.py
```
Our sphinx config file is another level down,